src/main/markdown/eco/application.md

```diff
@@ -0,0 +1,34 @@
+# Application Frameworks 
+
+One of the main tasks, when writing a GWT application is separating the code and managing the state. GWT offers with
+[MVP Activities and Places](/doc/latest/DevGuideMvpActivitiesAndPlaces.html) modules to do that. Over the
+time other implementations evolved. 
+
+## domino-mvp and domino-historie<a id="domino-mvp-historyt"></a>
+
+Domino-mvp is a small, simple and a very lightweight framework for building both simple and complex applications using
+GWT, it introduce the concept of routing using both tokens and events, also allows the developers to write a modular 
+application and shared components with any other domino application.
+
+For more information about domino-mvp, see: [domino-mvp at GitHub](https://github.com/DominoKit/domino-mvp)
+
+Domino-historie is a wrapper for the browser history state API.
+
+For more information about domino-history, see: [domino-history at GitHub](https://github.com/DominoKit/domino-history)
```

Comment:
I would prefer to use the website link as it hosts the documentation now instead of the directly sending to github, here

https://dominokit.com/solutions/domino-history/v1